File: JS/Fibonacci.js
var fibonacci = function(n) 
{
  if(n==1) 
  {
    return [0, 1];
  } 
  else 
  {
    var m = fibonacci(n - 1);
    m.push(m[m.length - 1] + m[m.length - 2]);
    return m;
  }
};

 console.log(fibonacci(10));

File: C/02_Fibonacci_number/fibonacci.c
#include <stdio.h>

int fibonacci(int n) {
    if (n <= 1) {
        return 1;
    } else {
        return fibonacci(n - 1) + fibonacci(n - 2);
    }
}

int main() {
    int SIZE = 12;
    for (int i = 0; i < 12; i++) {
        printf("%d ", fibonacci(i));
    }
   return 1;
}
